frontend/src/app/app.module.ts

```diff
@@ -11,35 +11,35 @@ import { CommonModule } from '@angular/common';
 import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
 import { NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
 import { MatBadgeModule } from '@angular/material/badge';
+import { MatButtonModule } from '@angular/material/button';
 import { MatButtonToggleModule } from '@angular/material/button-toggle';
+import { MatCardModule } from '@angular/material/card';
```

Comment:
URL: http://localhost:4200/settings/modelsources/t4c/instance/1

Issue: Gap between mat-cards is missing
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/541fb70f-2eef-41e9-b9dd-83b9710e0f95)